src/parkapi_sources/converters/goldbeck/converter.py

```diff
@@ -44,8 +44,8 @@ def __init__(self, *args, **kwargs):
             **goldbeck_header_rows,
         }
 
-    def map_row_to_parking_site_dict(self, mapping: dict[str, int], row: list[Cell]) -> dict[str, Any]:
-        parking_site_dict = super().map_row_to_parking_site_dict(mapping, row)
+    def map_row_to_parking_site_dict(self, mapping: dict[str, int], row: list[Cell], **kwargs: Any) -> dict[str, Any]:
```

Comment:
Here `kwargs` usage is fine, because you don't need `column_names`, so you can just use `kwargs` there